pinot-core/src/main/java/org/apache/pinot/core/segment/creator/StatsCollectorConfig.java

```diff
@@ -35,16 +36,19 @@
 public class StatsCollectorConfig {
 
   private final Schema _schema;
+  private final TableConfig _tableConfig;
   private final SegmentPartitionConfig _segmentPartitionConfig;
 
   /**
    * Constructor for the class.
    * @param schema Data schema
    * @param segmentPartitionConfig Segment partitioning config
    */
-  public StatsCollectorConfig(@Nonnull Schema schema, SegmentPartitionConfig segmentPartitionConfig) {
+  public StatsCollectorConfig(Schema schema, TableConfig tableConfig, @Nullable SegmentPartitionConfig segmentPartitionConfig) {
```

Comment:
(nit) Can we put table config in front of schema for consistency?

Reply:
sure , done